diff --git a/tools/dpdk-version.sh b/tools/dpdk-version.sh
index 0989aa4..75146cf 100755
--- a/tools/dpdk-version.sh
+++ b/tools/dpdk-version.sh
@@ -87,15 +87,19 @@ EOM
 	exit 1
 }
 
-if [ -z ${RTE_SDK} ] ; then
-	echo "*** RTE_SDK is not set, using "${PWD}
-	sdk=${PWD}
+if [ -n ${RTE_INCLUDE} ] ; then
+	fpath=${RTE_INCLUDE}/${fname}
 else
-	sdk=${RTE_SDK}
+	if [ -z ${RTE_SDK} ] ; then
+		echo "*** RTE_SDK is not set, using "${PWD}
+		sdk=${PWD}
+	else
+		sdk=${RTE_SDK}
+	fi
+
+	fpath=${sdk}/lib/librte_eal/common/include/${fname}
 fi
 
-fpath=${sdk}/lib/librte_eal/common/include/${fname}
-
 if [ ! -e ${fpath} ]; then
 	echo "File not found @ "${fpath}
 	usage
